views/project/invoice.erb

```diff
@@ -0,0 +1,126 @@
+<% @page_title = "#{@invoice_data[:name]} - Invoice" %>
+
+<% if @full_page %>
+  <div class="print-page"></div>
+<% else %>
+  <div class="space-y-1">
+    <%== render(
+      "components/breadcrumb",
+      locals: {
+        back: "#{@project_data[:path]}/billing",
+        parts: [
+          %w[Projects /project],
+          [@project_data[:name], @project_data[:path]],
+          ["Billing", "#{@project_data[:path]}/billing"],
+          ["#{@invoice_data[:name]} Invoice", "#"]
+        ]
+      }
+    ) %>
+    <%== render(
+      "components/page_header",
+      locals: {
+        title: "#{@invoice_data[:name]} Invoice",
+        right_items: [
+          "<a href='#{@project_data[:path]}/billing#{@invoice_data[:path]}?print=1' target='_blank' class='inline-flex items-center rounded-md bg-orange-600 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-orange-700 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-orange-600'>PDF</a>"
+        ]
+      }
+    ) %>
+  </div>
+<% end %>
+<div class="grid gap-6">
+  <!-- Invoice Card -->
+  <div
+    class="overflow-hidden rounded-lg shadow ring-1 ring-black ring-opacity-5 bg-white divide-y divide-gray-200 print:max-w-full print:w-full print:shadow-none print:ring-0"
+  >
+    <div class="px-4 py-5 sm:p-6">
+      <div class="flex justify-between">
+        <div>
+          <img class="h-10 w-auto" src="/logo-orange.png" alt="Ubicloud">
+          <address class="mt-4 not-italic text-gray-800">
```

Comment:
I think our address also should be part of invoice. If we change our address, addresses in the past invoices shouldn't change.

Reply:
I added `issuer_info` to invoice content